packages/warriorjs-core/src/getLevel.test.js

```diff
@@ -26,14 +26,14 @@ const levelConfig = {
       abilities: {
         walk: () => ({
           action: true,
-          description: `Moves one space in the given direction (\`'${FORWARD}'\` by default).`,
+          description: `Moves one space in the given direction (${FORWARD} by default).`,
```

Comment:
Let's add the `params` and `returns` info here so we can test they are added to the level JSON.